addons/mysql/templates/componentdefinition-mysql-5.7.yaml

```diff
@@ -0,0 +1,172 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ComponentDefinition
+metadata:
+  name: {{ include "mysql.componentDefName" . }}-5.7.44
+  labels:
+      {{- include "mysql.labels" . | nindent 4 }}
+spec:
+  provider: kubeblocks
+  description: mysql  compoent definition for Kubernetes
+  serviceKind: mysql
+  serviceVersion: 8.0.0
+
+  services:
+    - name: mysql-server
+      serviceName: mysql-server
+      spec:
+        ports:
+          - name: mysql
+            port: 3306
+            targetPort: mysql
+    - name: mysql
+      serviceName: mysql
+      generatePodOrdinalService: true
+      spec:
+        ports:
+          - name: mysql
+            port: 3306
+            targetPort: mysql
+  configs:
+    - name: mysql-replication-config
+      templateRef: oracle-mysql5.7-config-template
```

Comment:
`oracle-mysql` is for demo purpose. pls use configconstraint for mysql pls. check the name and replace them all (and better to ref the object of the same major version, do not mix them up).